app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class = "navbar">
+  <div class="nav-text mx-auto">
+    <h1 class="nav-title text-center"><%=link_to "Muncher.", root_path %></h1>
+    <%= form_tag(recipes_path, :method => "get") do %>
+      <p><%= label_tag(:recipe, "Find your flavor...") %></p>
+      <%= text_field_tag :recipe, params[:recipe], class: "input-word" %>
+      <%= submit_tag "Find", name: nil %>
+    <% end %>
+  </div>
+</div>
+
+</header>
+
+<% if @recipes %>
+<div class="container index-container mx-auto">
+  <% @recipes.each do |recipe| %>
+    <div class="card index-card bg-white">
+      <%= image_tag(recipe.image, alt: "recipe image", class: "card-img-top" ) %>
```

Comment:
I suggest making this image a link as well.  It's human nature to want to click on the images.